<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordDlg</class>
 <widget class="QFrame" name="RecordDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RecordDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
	font-family:&quot;Microsoft Yahei&quot;;
	font-size:13px;
	background-color:rgb(247,249,255);;
	color:black;
	min-height:30px;
	padding-left:10px;
	border:0px solid #cccccc;
	border-bottom-width:1px;
	border-top-width:0px;
}</string>
        </property>
        <property name="text">
         <string>录音文件列表，双击播放</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="trRecordList">
        <property name="styleSheet">
         <string notr="true">QWidget
{
	border:0px solid #cccccc;
	/*border-right-width:0xp;*/
	font-size:13px;
}

QTreeWidget::item
{
	min-height:30px;
	padding-left:10px;
	background-color:transparent;
}

 QTreeWidget::item:hover {
	background-color:rgba(100,100,100, 50);
 }

  QTreeWidget::item:selected {
	background-color:#cccccc;
	color:black;
 }</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="indentation">
         <number>0</number>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>250</number>
        </attribute>
        <column>
         <property name="text">
          <string>文件名</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="chkOpen">
           <property name="text">
            <string>打开环境音监控功能</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spnStopInterval">
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>分钟后自动停止录音</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSetting">
           <property name="text">
            <string>发送指令</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="bottom_bar">
     <property name="styleSheet">
      <string notr="true">QFrame#bottom_bar
{
	border:0px solid #cccccc;
	border-top-width:1px;
	border-bottom-width:1px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="lbStatus">
        <property name="text">
         <string>正在播放录音 ...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnStopRecord">
        <property name="text">
         <string>停止播放</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
